LandSandBoat/server
# Fix composure effect accuracy bonus calculation

**_I affirm:_**

## What does this pull request do?

Updates the scripts/effects/composure.lua to be able to cast the method getMainLevel() for calculating appropriate accuracy bonus, then applies and removes when appropriate.

Closes #6330 

## Steps to test these changes

Change job to RDM50 or greater
Use !getstats on yourself and note accuracy boost
Use job ability composure
Use !getstats on yourself and note accuracy boost

Your bonus accuracy should be roughly 1/2 of RDM Main level (35 at 70, 45 at 90, etc.)
